common/rootfs/usr/bin/get_arch

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+mode=${1:-unix}
+case $(arch) in
+    x86_64|amd64)
+      case $mode in
+          unix|qemu)
+            echo "x86_64";;
+          docker|ha)
+            echo "amd64";;
+      esac
+    ;;
+    aarch64|arm64)
+      case $mode in
+          unix|ha)
+            echo "aarch64";;
+          docker)
+            echo "arm64";;
+          qemu)
+            echo "arm-64";;
+      esac
+    ;;
+esac
```

Comment:
I am not a huge fan of this "centralized" arch management, another indirection to follow makes it harder to read/debug files, and since architecture are so different named everywhere, the next artefact/package quite certainly need another entry...

But I can live with it.